streebog/src/table.rs

```diff
@@ -3,6 +3,8 @@
 //!
 //! It was created using `gen_table` function
 
+#[cfg_attr(feature = "cargo-clippy", allow(unreadable_literal))]
```

Comment:
Not sure why you allow `unreadable_literal` here and in whirlpool, but in other places add underscores.

Reply:
This is what I should do in most other places.